package-lock.json

```diff
@@ -2045,11 +2039,6 @@
         "supports-color": "^5.3.0"
       }
     },
```

Comment:
Below here in the lock file, these updates seemed to implicitly happen from me removing the 3 libraries:

* apn
* base64url
* pem